README.md

```diff
@@ -268,6 +268,14 @@ Thus, these should all be set on the JVM command line using `-D`.
   Else, the JCA will get the implementation from another registered provider (usually stock JCE).
   Using JCE's impelmentation is generally recommended unless using ACCP as a standalone provider
   Callers can choose to register ACCP's implementation at runtime with a call to `AmazonCorrettoCryptoProvider.registerEcParams()`
+* `com.amazon.corretto.crypto.provider.registerSecureRandom`
+  Takes in `true` or `false` (defaults to `false` in FIPS mode, `true` in non-FIPS).
```

Comment:
also an excellent call-out. we need to be very explicit about this. will add.

Reply:
I agree, it does seem counterintuitive. Especially since we're being opinionated about the default state (as opposed to having registerSecureRandom always default to false). But perhaps it's worth clarifying that this property only affects the plumbing up to the Java layer. In FIPS mode, AWS-LC will continue to use its own FIPS-validated entropy source internally regardless of this property value.